wikimedia/wikipedia-ios
# Fix Apple Pay button tap gesture bug

**Phabricator:** https://phabricator.wikimedia.org/T365043

### Notes
More fixes to the donate form. This time the tap gesture recognizer for `WMFApplePayDonateButton` wasn't registering in iOS 18. There is now a SwiftUI Apple Pay Button called `PayWithApplePayButton` available to us since changing the minimum deployment target to iOS 16. Changing the element to this fixes the bug.

### Test Steps
1. Confirm Apple Pay button looks good in all themes.
2. Tap donate button in simulators for iOS 17 & 18. (Note, to test with iOS 16 simulator, I had to change the project deployment target down to iOS 16.4 from iOS 16.6, because I could not find any other iOS 16 simulator version). Confirm payment sheet displays.
